src/utils.rs

```diff
@@ -1,17 +1,51 @@
 use std::fs::File;
 use std::io::{Read, Write};
 use std::path::{Path, PathBuf};
+use std::env;
 
 use rustc_serialize::json;
+use git2::{Repository, Signature};
 
 use project_types::Project;
 use error::Error;
 
 
+/// Stages a file and commits it
+/// If a path to the repository is not given, assume it is in the current directory
+/// Impure.
+pub fn commit_file<P: AsRef<Path>>(
+    file_path: &Path, repo_path: Option<P>, signature: &Signature, msg: &str
+) -> Result<(), Error> {
+    
+    let repo = try!(get_repo_from_path(repo_path));
+
+    let _ = repo.index()
```

Comment:
Um, I think you did this while testing and forgot to change it. Are you sure you want to be discarding this result and returning an OK no matter what?